src/commands/dev/index.ts

```diff
@@ -50,6 +50,10 @@ export default async function main(ctx: NowContext) {
     });
     args = getSubcommand(argv._.slice(1), COMMAND_CONFIG).args;
     output = createOutput({ debug: argv['--debug'] });
+
+    if (argv['--debug']) {
+      process.env.NOW_BUILDER_DEBUG = '1';
```

Comment:
Let's add a code comment about why this is being set.

Reply:
@TooTallNate  I've added comment why `NOW_BUILDER_DEBUG` is needed.